tools/shared.py

```diff
@@ -2187,7 +2207,7 @@ def parse_symbols(output, include_internal=False):
         continue # e.g.  filename.o:  , saying which file it's from
       parts = [seg for seg in line.split(' ') if len(seg)]
       # pnacl-nm will print zero offsets for bitcode, and newer llvm-nm will print present symbols as  -------- T name
-      if len(parts) == 3 and parts[0] in ["00000000", "--------"]:
+      if len(parts) == 3 and parts[0] == "--------" or re.match(r'^\d+\d+\d+\d+\d+\d+\d+\d+$', parts[0]):
```

Comment:
So this is supposed to match any 8 digits? Do we need all the `+` (i.e. could it be `\d\d\d\d\d\d\d\d`)?

Reply:
Hmm, I think you're right, let's see what the tests say to be sure.

Reply:
Yeah, looks fine on CI. Not sure why I had those `+`s...